content/blog/2017-11-16-ropensci-changes.md

```diff
@@ -0,0 +1,104 @@
+
+Unfortunately, just like many a programming framework, Git can often be a teensy bit (read: extremely, prohibitively) intimidating, especially for beginners who don't need all of Git's numerous and baffling features.
+It’s one of those platforms that makes your life a million times better once you know how to use it, but if you’re trying to teach yourself the basics using the internet, or—heaven forbid—trying to untangle yourself from some Git-branch tangle that you’ve unwittingly become snarled in… (definitely done that one…) well, let’s just say using your knuckles to break a brick wall can sometimes seem preferable.
+Just ask the Git whizzes.
+They laugh, because they’ve been there, done that. 
+
+The funny thing is, doing basic version control in Git only requires a few commands.
+After browsing through the available project ideas and settling into teams, a group of eight of us made a list of the commands that we use on a daily basis, and the list came to about a dozen.
+We looked up our Git histories and compiled a Git vocabulary, which came out to less than 50 commands, including combination commands. 
+
+As [Nick Golding](https://github.com/goldingn) so shrewdly recognized in the lead up to this year’s unconference, the real obstacles for new Git users are (a) the scary, scary terminal window and (b) the fact that Git terminology was apparently chosen by randomly opening a verb dictionary and blindly pointing to a spot on the page.
+(Ok, I’m exaggerating, but the point is that the terminology is pretty confusing).
+We decided to address these two problems by making a package that uses the R console and reimagining the version control vocabulary and workflow for people who are new to version control and only need some of its many features.
+
+Somewhat ironically, nine people worked for two days on a dozen branches, using Git and GitHub to seamlessly merge our workflows.
+It was wonderful to see how so many people’s various talents can be combined to make something that no group members could have done all on their own. 
+
+Enter, `changes` ( [repo](https://github.com/ropenscilabs/ozrepro), [website](https://ropenscilabs.github.io/ozrepro/) – made using [pkgdown](https://github.com/hadley/pkgdown)), our new R package to do version control with a few simple commands.
+It uses Git and [Git2r](https://cran.r-project.org/web/packages/Git2r/index.html) under the hood, but new users don’t need to know any Git to begin using version control with changes.
+Best of all, it works seamlessly with regular Git. So if a user thinks they're ready to expand their horizons they can start using git commands via the [Githug](https://GitHub.com/jennybc/Githug) package, RStudio's git interface, or on the command line.
```

Comment:
a reason for using Git and git? or choose one capitalization

Reply:
I changed most of these gits to Git because that's what they use on [their website](https://git-scm.com/), even though I prefer the lower-case version.

Looks like I missed this one though - oops!

Reply:
agree. good to use Git in this case I think. 
I think I saw more than one git, so pls check